funcx_web_service/routes/funcx.py

```diff
@@ -784,10 +777,10 @@ def reg_function(user: User, user_uuid):
                 container=container
             )
 
-        if group:
+        if group_uuid:
```

Comment:
@ryanchard - this is the key change. When we insert the function record, we record the group uuid and save that string